components/pd_client/src/lib.rs

```diff
@@ -198,6 +198,15 @@ impl BucketStat {
         }
     }
 
+    pub fn ingest_sst(&mut self, key: u64, value_size: u64) {
```

Comment:
key may be renamed as key_size

Reply:
Actually from the caller code the 'key' parameter actually is the keys count. Then the code below is incorrect---it should not be write_bytes.
 for stat in self.stats.mut_write_bytes() {
            *stat += key;
        }

Reply:
good catch, fixed